README.md

```diff
@@ -211,11 +213,13 @@ function MyComponent() {
 
 ### `useFeatureFlags`
 
-The `useFeatureFlags` hooks returns an object of Flag identifier/Flag value pairs. You can pass an array of Flag
+The `useFeatureFlags` hook returns an object of Flag identifier/Flag value pairs. You can pass an array of Flag
```

Comment:
should `Flag` be de-capitalised?

Reply:
I had thought about that too, but decided to punt on it. We can address in a future PR.